custom_types.yaml

```diff
@@ -426,20 +426,28 @@ node_types:
       schedule:
         required: no
         type: string
-      name:
-        required: no
-        type: string
       description:
         required: no
         type: string
       retries:
         required: no
         type: integer
-      parents:
-        required: no
-        type: list
-        entry_schema:
-          type: string
+   capabilities:
+     - job_successor:
```

Comment:
@maricaantonacci you have to remove the "-" char before job_successor in the capabilities section.